Source/Scene/GroundPrimitive.js

```diff
@@ -74,6 +74,12 @@ define([
      * is supported at this time.
      * </p>
      * <p>
+     * Because of the cutting edge nature of this feature in WebGL, it requires the EXT_frag_depth extension, which is currently only supported in Chrome,
+     * Firefox, and Edge. Apple support is expected in iOS 9 and MacOS Safari 9. Android support varies by hardware and IE11 will most likely never support
```

Comment:
There's also no need to mention the Entity API either.  I would change this doc to something like:

> This feature requires the the WebGL EXT_frag_depth extension, which is supported in Chrome, Firefox, Safari, and Edge; but not IE 11. For mobile hardware, EXT_frag_depth support may also vary by device, you can use webglreport.com to verify support for your target hardware.

I think I tested it on Safari, but if you guys want to check again, go for it.